lib/AST/ASTContext.cpp

```diff
@@ -715,6 +714,10 @@ ASTContext::~ASTContext() {
   getImpl().~Implementation();
 }
 
+void ASTContext::SetPreModuleImportCallback(std::function<bool(llvm::StringRef ModuleName, bool IsOverlay)> callback) {
+  ASTContext::PreModuleImportCallback = callback;
```

Comment:
Why do you need to qualify `PreModuleImportCallback` here ?

Reply:
I don't, but I think I originally thought that `PreModuleImportCallback` was static even though it's not so I can remove the qualification.